src/lib/services/chat.ts

```
@@ -1,58 +1,172 @@
+			const testResult = await generateText({
+				model: openai('gpt-4o'),
+				prompt: 'Say "test"',
+				maxTokens: 5,
+			});
+
+			console.log('API key test successful:', testResult.text);
+		} catch (error) {
+			console.error('API key test failed:', error);
+			this.handleAiError(error);
+		}
+	} private handleAiError(error: any): never {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)